docs/how-to-translate-files.md

```diff
@@ -27,7 +12,7 @@ We just ask that you understand the following:
 
    Translating freeCodeCamp's resources is one of the most fun and rewarding experiences as a contributor, and it works best if you involve your friends and colleagues who speak the same world language as you.
 
-   We recommend joining [our community forum](https://forum.freecodecamp.org/c/contributors/3) and [contributors chat room](https://discord.gg/PRyKn3Vbay) with your friends and showing your interest before starting off with translations. Crowdin makes it easy to contribute translations, but it's still a lot of work.
+   You can start by reading [this announcement](https://www.freecodecamp.org/news/help-translate-freecodecamp-language/). We recommend joining [our community forum](https://forum.freecodecamp.org/c/contributors/3) and [contributors chat room](https://discord.gg/PRyKn3Vbay) with your friends and showing your interest before starting off with translations. Crowdin and other tools make it easy to contribute translations, but it's still a lot of work.
```

Comment:
The link to Discord does not do directly to the contributors channel. We can say "We recommend joining our community forum and Discord" instead, OR, we can use this direct link to the contributors channel:

https://discord.com/channels/692816967895220344/715074489422970962

IMHO, it's better to say Discord and not contributors chat room, so they can join the "Start Here" and then "introductions"

We should wait for this change/update. There are something been discussed right now in regard to the Discord channels naming convention.